src/static.ts

```diff
@@ -311,6 +319,8 @@ export interface ChatPrompt {
   escapedPrompt?: string;
   command?: string;
   context?: string[];
+  attachmentContent?: string;
+  inputText?: string;
```

Comment:
What is this in comparison to prompt?

Reply:
It is not so preferable to add these to typedef, if we're not planning to send it back to consumer. Since consumer uses this and they might expect to have an input text and attachment content. Instead, create a private interface extending from ChatPrompt for your use case